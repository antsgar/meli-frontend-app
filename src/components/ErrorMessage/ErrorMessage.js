import styles from "./ErrorMessage.module.scss";

const ErrorMessage = ({ message }) => {
    return <div className={styles.container}>{message}</div>;
};

export default ErrorMessage;
