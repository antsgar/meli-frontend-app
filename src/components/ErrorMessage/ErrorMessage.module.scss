@import "../../styles/vars";

.container {
    height: 100%;
    color: $text-color-default;
    font-size: 18px;
    display: flex;
    justify-content: center;
    align-items: center;
}