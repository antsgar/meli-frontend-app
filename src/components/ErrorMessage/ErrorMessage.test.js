import React from "react";
import { mount } from "enzyme";
import ErrorMessage from './ErrorMessage';

const message = "Error message";
let wrapper;

beforeEach(() => {
    wrapper = mount(<ErrorMessage message={message} />);
});

describe("ErrorMessage", () => {
    it("should render component", () => {
        expect(wrapper).toMatchSnapshot();
    });

    it("should show error message", () => {
        expect(wrapper.text()).toEqual(message);
    });
});