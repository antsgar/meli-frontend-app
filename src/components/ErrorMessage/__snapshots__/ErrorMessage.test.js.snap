// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should render component 1`] = `
<ErrorMessage
  message="Error message"
>
  <div
    className="container"
  >
    Error message
  </div>
</ErrorMessage>
`;
